qa/supply-chain/imports.lock

```diff
@@ -223,15 +192,6 @@ criteria = "safe-to-deploy"
 version = "1.0.2"
 notes = "This is a small crate which forbids unsafe code and is a straightforward implementation of the adler hashing algorithm."
 
-[[audits.bytecode-alliance.audits.arrayref]]
-who = "Nick Fitzgerald <fitzgen@gmail.com>"
-criteria = "safe-to-deploy"
-version = "0.3.6"
-notes = """
-Unsafe code, but its logic looks good to me. Necessary given what it is
-doing. Well tested, has quickchecks.
-"""
-
```

Comment:
We added an exemption for arrayref 0.3.8, when we could have relied on this audit plus vetting the diff for 0.3.6 -> 0.3.8. I've done that audit so I'll push it.